hw11/Lesson11.py

```diff
@@ -0,0 +1,64 @@
+import datetime
+import inspect
+
+func_name = None
+
+
+def my_decorator_func(func):
+    def deco_func():
+        func()
+        global func_name
+        frame = inspect.currentframe()
+        func_name = inspect.getframeinfo(frame).function
+        now_ = datetime.datetime.now()
+        print(f'The function name is {func_name} - was called at {now_.strftime("%H:%M")}')
+    return deco_func()
+
+
+@my_decorator_func
+def my_func():
+    print('This is my func')
+
+
+class ContextManager():
+    def __init__(self):
+        print('\nWhat is the sense of this task?')
+
+    def __enter__(self):
+        print('='*10)
+        # return self
+
+    def __exit__(self, exc_type, exc_value, exc_traceback):
+        print('='*10)
```

Comment:
You should check here if there is some exception and print its value: `if exc_type: print(exc_value)`
To avoid raising of the exception add `return True` to the end of the method

Reply:
I have added all changes, Oleksii, could you please review